src/runtime.js

```diff
@@ -54,13 +54,26 @@ define([
     /**
      * @constructor
      */
+    function RuntimeConfig() {
+        this._modeConfig = 'default';
```

Comment:
For states you can use constants like:

```
RuntimeConfig.MODES = {
       DEFAULT: 'default',
       TEST: 'test'
};
```

And you use to compare against these enumeration to make more maintainable

Reply:
I agree